parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoParquetWriter.java

```diff
@@ -43,6 +43,8 @@
    * @param blockSize            HDFS block size
    * @param pageSize             See parquet write up. Blocks are subdivided into pages for alignment and other purposes.
    * @throws IOException if there is an error while writing
+   *
+   * @deprecated will be removed in 2.0.0.; Used ProtoParquetWriter.Builder instead
```

Comment:
Please, also use the java annotation `@Deprecated` for all the deprecated methods.

Reply:
Got it, thanks for spotting that!